vortex/src/session.rs

```diff
@@ -45,6 +46,7 @@ impl Default for VortexSession {
             EncodingRef::new_ref(SequenceEncoding.as_ref()),
             EncodingRef::new_ref(SparseEncoding.as_ref()),
             EncodingRef::new_ref(ZigZagEncoding.as_ref()),
+            EncodingRef::new_ref(PcoEncoding.as_ref()),
```

Comment:
nit: the list is currently alphabetized, could you put PCoEncoding just before RunEndEncoding?

Reply:
:+1:

Reply:
I don’t think we need a compile time flag. Until we wire it into compressor adding an encoding doesn’t do anything to existing users.